### DISCUSSION TITLE
What’s up for LNUG in 2024

### DISCUSSION THREAD

**Discussion by admataz** [lnug/feedback#121] [Ideas]
I think we're all glad to be back to regular meet-ups this year - and we've had some good events. What shall we do in the coming 12 months? Would love to get some ideas from everyone.

**Comment by twofizzy** [comment]
I really enjoyed the latest meeting with Ben and Brian.
Ben's was more inspiring in terms of enabling peer to peer sharing and collaboration, which I think is more and more needed in this time.
Brian's optimisation using streaming, I'm super excited to implement later on in my projects, with large data-sets. I feel like it will really give me the edge when ti comes to making site that I produce, run quickly and smoothly.

Thanks for putting this event together!

I've really been looking to be inspired recently with new ideas and feel like this has done it for me!

**Reply by twofizzy** [reply]
Putting this out there! I'd be interested in doing a mini-hackathon or coding ideas around a topic/technology/principle and presenting them at the next meetups? I feel like I'd really learn a lot from this in a more applicable way. I'd love to know thoughts about this?
